src/app/general-style/general-style.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, forwardRef, Input } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+// import { stylesSheet_Input } from '../../shared/style.sheets';
+// import { valueDefault } from '../../shared/value.sheets';
+
+@Component({
+  selector: 'general-style-component',
+  templateUrl: './general-style.component.html',
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => GeneralStyleComponent),
+      multi: true,
+    },
+  ],
+})
+export class GeneralStyleComponent {
+  @Input() prop;
+  @Input() nameProp;
+  @Input() forControlAcc;
+
```

Comment:
add types and full name of properties
```suggestion
  @Input() prop;
  @Input() nameProp;
  @Input() forControlAcc;

```